<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tayarni - Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pushButton_LoginL">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: white;
border-radius: 15px;
border: 2px solid rgb(1, 38, 56);
background-color: rgba(255, 255, 255,100);
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(167, 218, 239), stop:1 rgb(47, 109, 139));
}</string>
   </property>
   <property name="text">
    <string>Se connecter</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="LoginInfoUser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>571</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border-radius: 15px;
border: 2px solid rgb(1, 38, 56);
background-color: rgba(47, 109, 139,50);
}
QPushButton {
border-radius: 15px;
border: 2px solid rgb(1, 38, 56);
background-color: rgba(47, 109, 139,50);
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_PasswordL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-radius: 15px;
border: 2px solid rgb(1, 38, 56);
background-color: rgba(255, 255, 255,100);
}
QLineEdit:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(167, 218, 239), stop:1 rgba(47, 109, 139,50));
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_UsernameL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-radius: 15px;
border: 2px solid rgb(1, 38, 56);
background-color: rgba(255, 255, 255,100);
}
QLineEdit:hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(167, 218, 239), stop:1 rgba(47, 109, 139,50));
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_UsernameL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
border: 2px solid rgb(1, 38, 56);
background-color: rgba(255, 255, 255,100);</string>
    </property>
    <property name="text">
     <string>Utilisateur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PasswordL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
border: 2px solid rgb(1, 38, 56);
background-color: rgba(255, 255, 255,100);</string>
    </property>
    <property name="text">
     <string>Mot de Passe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gif">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="showpassword">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/eye.png);

}
QPushButton {
border-radius: 15px;
border: 2px solid rgb(1, 38, 56);
background-color: rgba(47, 109, 139,50);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/tayarni.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="oublier">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: white;
border-radius: 15px;
border: 2px solid rgb(1, 38, 56);
background-color: rgba(255, 255, 255,100);
}
QPushButton:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgb(167, 218, 239), stop:1 rgb(47, 109, 139));
}</string>
   </property>
   <property name="text">
    <string>Mot de Passe Oubli√©</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
